lua/barbar/render.lua

```diff
@@ -34,23 +33,29 @@ local JumpMode = require'barbar.jump_mode'
 local Layout = require'barbar.layout'
 local state = require'barbar.state'
 local utils = require'barbar.utils'
+-- local fs = require'barbar.fs' -- For debugging purposes
 
 --- Last value for tabline
 --- @type string
 local last_tabline = ''
 
 --- Concatenates some `groups` into a valid string.
 --- @param groups barbar.render.group[]
+--- @param text_only? boolean
 --- @return string
-local function groups_to_string(groups)
+local function groups_to_string(groups, text_only)
```

Comment:
Should the text_only param just be a different function? i.e. `groups_to_string_raw`? That way we don't have to do an `if` for every iteration of `groups` just to determine the concatenation to `result` (since `text_only` is only ever true for debug code)